js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navMenu = document.querySelector('nav');
+
+let newA1 = document.createElement('a');
+newA1.textContent = 'Cat Pictures';
+navMenu.appendChild(newA1);
+
+let navItems = document.querySelectorAll('nav > a')
+navItems[0].textContent = siteContent["nav"]['nav-item-1'];
+navItems[1].textContent = siteContent["nav"]['nav-item-2'];
+navItems[2].textContent = siteContent["nav"]['nav-item-3'];
+navItems[3].textContent = siteContent["nav"]['nav-item-4'];
+navItems[4].textContent = siteContent["nav"]['nav-item-5'];
+navItems[5].textContent = siteContent["nav"]['nav-item-6'];
+
+let newA3 = document.createElement('a');
+newA3.textContent = 'Affiliate Program';
+navMenu.prepend(newA3);
+
+let navItemsFull = document.querySelectorAll('nav > a')
+navItemsFull.forEach(x => x.style.color = 'green')
+
+let headerImage = document.querySelector('#cta-img');
+headerImage.src = siteContent["cta"]["img-src"]
+
+let middleImage = document.querySelector('#middle-img')
+middleImage.src = siteContent["main-content"]["middle-img-src"]
+
+let ctaHeader = document.querySelector('.cta-text > h1')
+
+let text1 = document.createTextNode('dom')
```

Comment:
Another way to each on a seperate line would be to split the words into an array, use forEach to create a div and during each iteration, set the textcontent of the div to this specific word. This will put the words a separate because are div are block elements.